packages/web-components/fast-components/src/scroller/scroller.definition.ts

```diff
@@ -0,0 +1,51 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/esm/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+
+export const fastScrollerDefinition: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fast-scroller",
+            title: "Scroller",
+            description: "The FAST scroller element",
+            attributes: [
+                {
+                    name: "view",
+                    description: "Default or mobile view",
```

Comment:
Can this have the list of allowable values?

Reply:
Updated to show the values.